vizro-core/src/vizro/models/_controls/filter.py

```diff
@@ -35,12 +35,28 @@
     "temporal": (DatePicker,),
 }
 
+ALLOWED_SELECTORS = {
+    "numerical": SELECTORS["numerical"] + SELECTORS["categorical"],
+    "temporal": SELECTORS["temporal"] + SELECTORS["categorical"],
+    "categorical": SELECTORS["categorical"],
+}
+
```

Comment:
Can we just modify `SELECTORS` to be this? Having a separate `SELECTORS` and `ALLOWED_SELECTORS` feels confusing - if we really need it then please add a comment explaining why 🙂 

Also would be good to link to my comment in this PR explaining here why we want categorical selectors in all the different types so we can refer back to it in future.

```suggestion
CATEGORICAL_SELECTORS = (Dropdown, ...)
SELECTORS = {
    "numerical": (RangeSlider, Slider, *CATEGORICAL_SELECTORS),
    "temporal": ...,
    "categorical": ...,
}
```

Reply:
I added the comment from the screenshot inside the `filter.py`:
<img width="882" alt="image" src="https://github.com/mckinsey/vizro/assets/108530920/f9fad1e2-5686-40b8-ae54-6dc53e5743d9">

The link from the comment (https://github.com/mckinsey/vizro/pull/319#discussion_r1524888171) actually points to the discussion we had in this PR above.